stackstac/to_dask.py

```diff
@@ -161,6 +163,8 @@ def asset_entry_to_reader_and_window(
 def fetch_raster_window(
     asset_entry: Optional[Tuple[Reader, windows.Window]],
     slices: Tuple[slice, ...],
+    window_dtype: np.dtype = np.dtype("float64"),
```

Comment:
Using the keyword `window_dtype` to avoid a conflict with `blockwise` using `dtype` as well.